BUILD.bazel

```diff
@@ -85,6 +86,24 @@ go_image(
     x_defs = {"main.gitCommit": "{STABLE_GIT_COMMIT}"},
 )
 
+# Initialize the default /data storage directory since the bazel-remote
+# user does not have sufficient permissions at the root (/) directory.
+# Since an empty directory cannot be packaged by the pkg_tar rule, create
+# the "EMPTY" file.
+genrule(
+    name = "empty",
+    srcs = [],
+    outs = ["EMPTY"],
+    cmd = "touch $@",
+)
```

Comment:
Perhaps we can use this instead?
https://github.com/bazelbuild/rules_docker/issues/37#issuecomment-544652904

Reply:
Using `empty_dirs` builds without errors, but the directories are owned by the root user, and not the `user` attribute (in this case "bazel-remote").

Here is the error upon running the built image when using `empty_dirs`:
```12:45:41 ⌂91% [mmikitka:~/src/bazel-remote] docker-nonroot(+4/-26) 8s ± docker run --rm -it bazel:bazel-remote-image
2020/07/03 16:45:49 bazel-remote built with go1.14.3 from git commit 63bcf072dd5f197e3ee539d4ec32062c9e583d74.
2020/07/03 16:45:49 Initial RLIMIT_NOFILE cur: 1048576 max: 1048576
2020/07/03 16:45:49 Setting RLIMIT_NOFILE cur: 1048576 max: 1048576
2020/07/03 16:45:49 mkdir /data/cas: permission denied
```